### DISCUSSION TITLE
Admin UI: Delete button and bulk

### DISCUSSION THREAD

**Discussion by gonzamoiguer** [payloadcms/payload#627] [Feature Requests & Ideas]
One small feature I do miss from other CMS is the ability to delete and bulk delete documents from the collections view in the admin. I think it would add to the general UX and usability. If you have to delete 10 entries, the process becomes quite tedious.

Hope to see this feature sometime. Thanks in advance and congrats to everyone involved on this amazing product.

**Comment by arieltonglet** [comment]
I second that, and would love to see **bulk delete** as a core feature!

Still on the bulk department, another feature that I think would be nice is to let the user **bulk upload** and create multiple items in a **upload collection** (ex: uploading several images at once, instead of having to create item/upload one by one)

**Comment by JesseRigon** [comment]
I've heard the team say on calls that they are not really for creating a cli for payload which I disagree with. I'm not a fan of php so I don't like wordpress but their cli is great for programmatically doing things like bulk modifications, cron jobs and remote ssh. this would honestly solve the bulk thing and allow for a lot of power user functionality. I know it's a whole other topic, just wanted to say this as often as possible from now on, when applicable, to change their minds.
